src/scanner.cc

```diff
@@ -213,10 +245,8 @@ struct Scanner {
           next();
 
           if (is_identifier_start_char(peek())) {
-            if (did_advance) {
-              set(HEREDOC_BODY);
-            }
```

Comment:
The `set` was behind and if-statement so that when debugging, a `set SYMBOL` line is only printed when there is a successful match. Now that we print the return value of this function call, we can just set this symbol (it doesn't affect outcome of the scanner when `false` is returned).